shaders/src/common_graphics.fs

```diff
@@ -24,6 +24,22 @@ void unpremultiply(inout vec4 color) {
     color.rgb /= max(color.a, FLT_EPS);
 }
 
+/**
+ * Applies a full range YCbCr to sRGB conversion and returns an RGB color.
+ *
+ * @public-api
+ */
+vec3 ycbcrToRgb(float luminance, vec2 cbcr) {
+    // Taken from https://developer.apple.com/documentation/arkit/arframe/2867984-capturedimage
+    mat4 ycbcrToRgbTransform = mat4(
```

Comment:
You can make the matrix const just for completeness. You should also get rid of the + signs and f suffixes just for consistency with the rest of our shaders.